### ISSUE TITLE
Invalid indentation level in kubectl describe node output

### ISSUE THREAD

**Issue by dty1er** [kubernetes/kubernetes#95638]
**What happened**:

When executing "kubectl describe node", indentation level of items in "Capacity", "Allocatable" and "System Info" is not consistent with others. Although other parts have 2 spaces, they only have one space.

![image](https://user-images.githubusercontent.com/60682957/96226149-29448200-0fcd-11eb-8790-69c70e7b09d4.png)

**What you expected to happen**:

Indentation with 2 spaces.

**How to reproduce it (as minimally and precisely as possible)**:

Run "kubectl describe node" for an arbitrary cluster.

**Anything else we need to know?**:

N/A

**Environment**:
- Kubernetes version (use `kubectl version`): v1.19.2. Additionally, kubectl version is v1.16.3 but I don't think it's fixed in the latest one.
- Cloud provider or hardware configuration: minikube
- OS (e.g: `cat /etc/os-release`): macOS
- Kernel (e.g. `uname -a`): Darwin PC2350 19.6.0 Darwin Kernel Version 19.6.0: Mon Aug 31 22:12:52 PDT 2020; root:xnu-6153.141.2~1/RELEASE_X86_64 x86_64
- Install tools: Docker

**Comment by brianpursley** [comment]
This was [fixed in 1.17.0](https://github.com/kubernetes/kubernetes/commit/4ce16254f7b89324dcbb43de70d0d95cb9ce5865)

version 1.16.9 output: 
```
System Info:
 Machine ID:                 caf633cddf734214bb5c2c8cb1aed0e9
 System UUID:                bdaa3a24-37ef-11e3-872e-90b40d9a1000
 Boot ID:                    16875b9f-425a-4125-ae50-f043b9db3444
 Kernel Version:             5.4.0-48-generic
 OS Image:                   Ubuntu 20.04 LTS
 Operating System:           linux
 Architecture:               amd64
 Container Runtime Version:  docker://19.3.8
 Kubelet Version:            v1.20.0-alpha.0-636-g6d001ebb68efd8.dev-1598402651
 Kube-Proxy Version:         v1.20.0-alpha.0-636-g6d001ebb68efd8.dev-1598402651
```

version 1.17.3 output:
```
System Info:
  Machine ID:                 caf633cddf734214bb5c2c8cb1aed0e9
  System UUID:                bdaa3a24-37ef-11e3-872e-90b40d9a1000
  Boot ID:                    16875b9f-425a-4125-ae50-f043b9db3444
  Kernel Version:             5.4.0-48-generic
  OS Image:                   Ubuntu 20.04 LTS
  Operating System:           linux
  Architecture:               amd64
  Container Runtime Version:  docker://19.3.8
  Kubelet Version:            v1.20.0-alpha.0-636-g6d001ebb68efd8.dev-1598402651
  Kube-Proxy Version:         v1.20.0-alpha.0-636-g6d001ebb68efd8.dev-1598402651
```

**Comment by dty1er** [comment]
I was using old kubectl and confirmed the problem is fixed in the latest kubectl. Let me close the issue.
